pkg/client/ocm/client.go

```diff
@@ -496,31 +514,43 @@ func (c client) GetQuotaCosts(organizationID string, fetchRelatedResources, fetc
 		return nil, err
 	}
 	quotaCostList := quotaCostResp.Items()
-	return quotaCostList, nil
+
+	if !fetchRelatedResources || len(relatedResourceFilters) == 0 {
```

Comment:
I think the behavior of this method is incorrect now:

In the case where fetchRelatedResources is `true` and relatedResourceFilters is empty(or nil) we are now returning a nil slice. That would be incorrect. The result should be all the QuotaCosts.

Can we also cover that in a test to ensure we don't break it in the future?

Reply:
edit: I was not right on my comment. In that case this if would be `true` and it would return the quotas as they are

Reply:
I'll add a test for this.